packages/lab/src/__tests__/__e2e__/tabs-next/TabsNext.cy.tsx

```diff
@@ -0,0 +1,417 @@
+const { Default: DefaultTabstrip, SimpleTabstrip } =
+  composeStories(tabstripStories);
+
+describe("Given a Tabs", () => {
+  describe("WHEN uncontrolled", () => {
+    describe("WHEN no defaultActiveTabIndex is provided", () => {
+      it("THEN first tab is selected", () => {
+        cy.mount(<DefaultTabs width={400} />);
+        cy.findAllByRole("tab")
+          .eq(0)
+          .should("have.attr", "aria-selected", "true");
+        cy.findByText("Content for Home tab");
+      });
+    });
+    describe("WHEN a defaultActiveTabIndex is provided", () => {
+      it("THEN the defaultActiveTabIndex is selected", () => {
+        cy.mount(<DefaultTabs defaultActiveTabIndex={1} width={400} />);
+        cy.findAllByRole("tab")
+          .eq(1)
+          .should("have.attr", "aria-selected", "true");
+        cy.findByText("Content for Transactions tab");
+      });
+    });
+  });
+  describe("WHEN controlled", () => {
+    describe("WHEN a activeTabIndex is provided", () => {
+      it("THEN the activeTabIndex is selected", () => {
+        cy.mount(<DefaultTabs activeTabIndex={1} width={400} />);
+        cy.findAllByRole("tab")
+          .eq(1)
+          .should("have.attr", "aria-selected", "true");
+        cy.findByText("Content for Transactions tab");
+      });
+    });
+    describe("WHEN a onActiveChange is provided", () => {
+      it("THEN the activeTabIndex is selected", () => {
+        cy.mount(
+          <DefaultTabs
+            onActiveChange={cy.spy().as("onActiveChange")}
+            activeTabIndex={1}
+            width={400}
+          />
+        );
+        cy.findAllByRole("tab")
+          .eq(1)
+          .should("have.attr", "aria-selected", "true");
+        cy.findByText("Content for Transactions tab");
+        cy.findAllByRole("tab").eq(0).click();
+
+        cy.get("@onActiveChange")
+          .should("have.been.calledOnce")
+          .should("have.been.calledWith", 0);
+      });
+    });
+  });
+});
+
+describe("Given a Tabstrip", () => {
+  describe("WHEN uncontrolled", () => {
+    describe("WHEN no defaultActiveTabIndex is provided", () => {
+      it("THEN first tab is selected", () => {
+        cy.mount(<DefaultTabstrip width={400} />);
+        cy.findAllByRole("tab")
+          .eq(0)
+          .should("have.attr", "aria-selected", "true");
+      });
+    });
+    describe("WHEN a defaultActiveTabIndex is provided", () => {
+      it("THEN the defaultActiveTabIndex is selected", () => {
+        cy.mount(<DefaultTabstrip defaultActiveTabIndex={1} width={400} />);
+        cy.findAllByRole("tab")
+          .eq(1)
+          .should("have.attr", "aria-selected", "true");
+      });
+    });
+    describe("WHEN a defaultActiveTabIndex is null", () => {
+      it("THEN no tab is selected", () => {
+        cy.mount(<DefaultTabstrip defaultActiveTabIndex={null} width={400} />);
+        cy.findAllByRole("tab")
+          .eq(0)
+          .should("have.attr", "aria-selected", "false");
```

Comment:
either the description is wrong or the test. I'd go with the test 😄

Reply:
Because I am only checking the first tab? I should probably loop through all tabs and check none are selected. I naively checked only the first since that is what usually gets auto-selected 😅